### ISSUE TITLE
Leaving -- because I am "banned" now

### ISSUE THREAD

**Issue by CaptainPRICE** [wiremod/wire#1941]
Wiremod, you have disrespected me (including my work).
I won't tolerate it anymore, especially from people like @adosikas (who constantly used to mock/troll me for *Boolean* stuff, something that's past behind, also derailing my announcement/topic every single time), and generally because I was being underestimated in the community.
Take my leave. I wish you all have a good day, and good luck.

P.S. There are certain exploits in Wire code (which I have found recently, but didn't have time to report them), so those won't be reported.
🖕

**Comment by oldmud0** [comment]
I might need some fact-checking on this, but do realize how many times this kind of action has been done by the Wiremod admins in the past decade: zero. There was an opportunity at every step of the way to apologize, but you did not, and even at this critical moment, you stick out the middle finger. It was not for no reason that the admins made this decision.

You are still welcome to submit patch requests, but I think your additions-oriented perspective on E2 deserves its own addon. Remember that demonstrating working code and results is almost always better than bickering about design and _claiming_ that you have working code, as baseless claims (especially regarding your accomplishments) have eroded the value of your word.

If bypassing the Wiremod development process facilitates your ability to develop what you want for Wiremod, so do it - the code base is right in front of you, and many of your past pull requests have been approved. Do not complain that your code is being rejected when what you have written have mostly been new bugs to track (potentially meaning work for other people to do for you), and not actual patches. If you took care to propose patches for even half of the 120 issues you made over the last three years, you could have become a developer by now.

**Comment by CaptainPRICK** [comment]
@oldmud0 
You got it all wrong. This has nothing to do with my requests/work made here.
I am leaving because people like gohidas had crossed their *line*, and that was enough already (accounting for the same past with him too).
It is about my proof-of-concept and side-projects Wire-related work (see `#contraptions` on Wire Discord for the most recent one, i hope he didn't delete his messages, if chat is still there you can read it).
His disrespect (i.e. calling me "troll", then derailing the topic) towards my recent PoC work is the **main reason of my leave** - nothing else.

**Comment by thegrb93** [comment]
If you ask me, you got a good dose of your own medicine.

**Comment by Anticept** [comment]
Blocking you from github happened because of this childish attitude that you've just proven, you originally weren't banned here, it was just going to be on discord.

I really tried with you. Gohidas at least stepped forth and admitted his mistakes, and was making efforts to be better, which is why he's being forgiven. Many times in the past, I've dealt with users in cases like this, and even dealt with some developer infighting back in 2008 or 2009. When things got out of line, I would mediate until it calmed down as long as progress was being made. Sometimes it took several times before it set in.

I tried really hard to show you we'd give you the same courtesy, but you are showing me you aren't interested in being _better_, you just want to be _right_. I can't keep that behavior around if there won't even be attempts. We're /extremely/ lenient, this really is the first time I've had to boot someone like this.
